R/rcpp.R

```diff
@@ -51,3 +84,52 @@ use_src <- function() {
 
   invisible()
 }
+
+set_makevars <- function(path) {
```

Comment:
This feels very fragile to me, and I don't want to maintain a function that attempts to modify any existing makevars. I think it would be better to have a simple path when makevars doesn't exist, and print instructions to the user when it does.